aspnetcore/fundamentals/metapackage.md

```diff
@@ -33,3 +36,35 @@ You can use the package trimming process to remove packages that you don't use.
 The following *.csproj* file references the `Microsoft.AspNetCore.All` metapackage for ASP.NET Core:
 
 [!code-xml[](../mvc/views/view-compilation/sample/MvcRazorCompileOnPublish2.csproj?highlight=9)]
+
+::: moniker range=">= aspnetcore-2.1"
+# Microsoft.AspNetCore.All metapackage for ASP.NET Core 2.1 and later version range locking
+
+`Microsoft.AspNetCore.All` 2.1 and later specifies package dependency ranges. The package dependency ranges ensure that only the excact version specified for each dependency is allowed. Any other dependency that attempts to lift a version for a component beyond what is specified in the meta-package will result in a NuGet warning, which the Microsoft.NET.Sdk will elevate to an error by default. 
```

Comment:
This is not true, there is no version locking in Microsoft.AspNetCore.All. Version locking is only added to Microsoft.AspNetCore.All.